### PR TITLE
[Week 10] Migrate to gymnasium

### PR DESCRIPTION
**Pull request by laktionov**
- Support new interfaces in seminar notebook
- Change `done` to `terminated or truncated` to iterate over env

### CODE REVIEW

**Comment by kharitonov-ivan** [comment]
Hi! Thx for PR. Let me see.

**Review by dniku — COMMENTED** [review]
Same question as for #533: why switch from v0 to v1?

**Comment by laktionov** [comment]
Hi, in this case `CartPole-v1` with `max_episode_steps=200` is equivalent to `CartPole-v0` and the update has been made in order not to face with an annoying warning.

**Review by dniku — APPROVED** [review]
Thanks! LGTM from me, but @kharitonov-ivan also wanted to take a look.

File: week10_planning/seminar_MCTS.ipynb [hunk 1]
```diff
@@ -71,6 +71,7 @@
     "import sys, os\n",
```
Comment (kharitonov-ivan) [inline week10_planning/seminar_MCTS.ipynb:71]: <pre class="ql-syntax" spellcheck="false">Line #90.                ,
</pre><p>I suggest clarifying briefly how the 'is_done' variable is calculated, whether from truncated or terminated states.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/yandexdataschool/Practical_RL/pull/534/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

**Comment by kharitonov-ivan** [comment]
@laktionov I've looked it over, added a few comments, otherwise all good.

**Review by kharitonov-ivan — APPROVED** [review]
👍

**Comment by dniku** [comment]
Thanks!

**Pull request merged**